Misc/NEWS.d/next/Library/2022-04-10-17-12-23.gh-issue-91230.T1d_fG.rst

```diff
@@ -0,0 +1,3 @@
+`warnings.catch_warnings` now accepts arguments for `warnings.simplefilter`,
```

Comment:
```suggestion
:func:`warnings.catch_warnings` now accepts arguments for :func:`warnings.simplefilter`,
```

This should fix the docs CI failure (you might have to reflow the paragraph a little to keep lines <80 characters)

Reply:
Done - thanks for the review Alex 😁